scripts/migrate.js

```diff
@@ -0,0 +1,33 @@
+var spawn = require('cross-spawn');
+var BPromise = require('bluebird');
+var fs = BPromise.promisifyAll(require('fs'));
+var path = require('path');
+
+var db = process.argv[2] || 'webmaker_testing';
+console.log( __dirname );
+fs.readdirAsync('./migrations').each(function(filename) {
+  return new BPromise(function(resolve, reject) {
+    var create = spawn('psql', ['-d', db, '-f', path.join( './migrations', filename)], {
```

Comment:
Feels like it makes more sense to rely on the information that's in `.env`, to parse the `POSTGRE_CONNECTION_STRING` var for the relevant data, including username and password if they're used. Either that, or at least allow the user to specify a username and password on the command line.